### ISSUE TITLE
New permissions?

### ISSUE THREAD

**Issue by Efreak** [Xlythe/Calculator#31]
As a user of the testing version on Google play, I just got a notification that this app wants new permissions.
Suddenly this looks less like a calculator and more like spyware. In one update, you've added internet access, location reporting, account/data access, and filesystem access. And yet, your change notes fail to mention a reason for _any_ of these. As someone concerned with privacy, I would appreciate some kind of explanation.

Looking at your repository, it would be nice to see your default branch switched over to the only active branch you have, and it would be even nicer to see downloadable releases (or at least tags that match your play store releases) and a changelog.

**Comment by Xlythe** [comment]
Discussion on G+: https://plus.google.com/+FeiKuan/posts/GVADRov8deB
tldr; I don't know why those permissions are being requested. It's considered a low priority bug.

There will be tagged releases when the app is officially released. Likewise, the default branch will switch then as well.

**Comment by Xlythe** [comment]
Fixed in latest commit. Play Services was adding the permissions -- not sure why I even added Play Services. Removed it.

**Comment by Efreak** [comment]
OK. Thank you _very_ much. I can understand advertising I see in other apps (I've seen ads in _some_ open source apps before), but some of these permissions just seem incredibly intrusive for that.

As far as the reasoning behind it goes? If google's build process is giving such permissions simply as a result of the minsdkversion or targetsdkversion, I find this unacceptable. It means that the permissions are useless are a means of identifying apps that can access private information and send it out online. The requirement for adding the internet permission to apps that just want advertisements is bad enough, as it makes it difficult to tell which apps might be leaking data.

Apologies for being so rude about it.

**Comment by Xlythe** [comment]
No worries. It happened because I was requesting Play Services (Google's library for maps, analytics, Android Wear, and a whole heck of a lot of other stuff). I was probably playing around with Android Wear at the time -- I honestly don't remember.

It also happens if you target Android 1-3 (which are 6 years old now, and gave those permissions for free). But that wasn't the case this time.
